active.cabal

```diff
@@ -1,5 +1,5 @@
 name:                active
-version:             0.3
+version:             0.4
```

Comment:
Can we just revert this version bump?  (For complicated reasons) `active-0.3` has never been released, so there is no need to bump the version.

Reply:
Done!